docs/en/API/Swagger-Integration.md

```diff
@@ -81,6 +81,22 @@ public override void OnApplicationInitialization(ApplicationInitializationContex
 }
 ```
 
+### Hide ABP Endpoints on Swagger UI
+
+To hide ABP-related endpoints, we can use the `HideAbpEndpoints` method when configuring Swagger options. If you want to hide ABP endpoints, add the following line into your Swagger Configuration:
```

Comment:
It is repeating the same sentence :) I would suggest: "If you want to hide the ABP's default endpoints, call the `HideAbpEndpoints` method in your Swagger configuration as shown in the following example:"